### PR TITLE
[MRG] Improving the documentation and adding CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sappelhoff**
After working on the readthedocs page (which now works, yey) I made some additional changes to be more open to contributors (`CONTRIBUTING.md`) etc.

Also: Referring to the stable version of the docs now ... There is still some issue with "latest" ... I don't know why :-) 

Ready to be merged from my side.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Contributions
+-------------
+
+Contributions are welcome in the form of pull requests.
```
Comment (choldgraf) [inline CONTRIBUTING.md]:
```suggestion
 Contributions are welcome in the form of feedback and discussion in issues,
 or pull requests for changes to the code.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Once the implementation of a piece of functionality is considered to be bug
```
Comment (choldgraf) [inline CONTRIBUTING.md]:
```suggestion
 Once the implementation of a piece of functionality is considered to be a bug
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+free and properly documented, it can be incorporated into the master branch.
+
+To help developing `pybv`, you will need a few adjustments to your
+installation as shown below.
+
+##### Install the development version
```
Comment (choldgraf) [inline CONTRIBUTING.md]: Here don't you think we should suggest people fork and clone rather than install the organizational repo?

**Comment by sappelhoff** [comment]
cool, all addressed :-)

Feel free to merge.

**Comment by choldgraf** [comment]
wooo! thanks!

**Pull request merged**